omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -234,6 +234,20 @@
 default['private_chef']['opscode-erchef']['couchdb_max_conn'] = '100'
 default['private_chef']['opscode-erchef']['ibrowse_max_sessions'] = 256
 default['private_chef']['opscode-erchef']['ibrowse_max_pipeline_size'] = 1
+# general search settings used to set up chef_index
+default['private_chef']['opscode-erchef']['search_provider'] = "solr" # solr, TODO cloudsearch
+default['private_chef']['opscode-erchef']['search_queue_mode'] = "rabbitmq" # rabbitmq, batch, or inline
```

Comment:
Definitely need some documentation here.  the max_size is actually the byte_size() of the document we are going to post.  This is a forward-looking feature since cloudsearch has a hard limit on the size of its updates.

The max_wait time is in milliseconds.  We should also document that it is the "max_wait" (1) for how long something will sit in the queue before initializing the post and (2) it is a "max" in the same sense that that the bus schedule tells you when the bus is supposed to come.